edgelet/workload/src/models/identity_certificate_request.rs

```diff
@@ -0,0 +1,45 @@
+/*
+ * IoT Edge Module Workload API
+ *
+ * No description provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)
+ *
+ * OpenAPI spec version: 2018-06-28
+ *
+ * Generated by: https://github.com/swagger-api/swagger-codegen.git
+ */
+
+#[allow(unused_imports)]
+use serde_json::Value;
+
+#[derive(Debug, Default, Serialize, Deserialize)]
+pub struct IdentityCertificateRequest {
+    /// Certificate expiration date-time (ISO 8601)
+    #[serde(
+        rename = "expiration",
```

Comment:
Is this `rename` redundant?

Reply:
This was auto code generated, not sure if this should be changed.

Reply:
Marking as resolved, unless you let me know otherwise.